apps/AppWithWearable/lib/main.dart

```diff
@@ -22,6 +23,15 @@ void main() async {
   await SharedPreferencesUtil.init();
   await MixpanelManager.init();
   await ObjectBoxUtil.init();
+  if (Env.oneSignalAppId != null) {
```

Comment:
Let's login independently of permissions, and let's not ask for permissions when the app opens, currently notification permissions are being asked at home/page.dart

Reply:
I just now actually made this change and after pushing it, realised that the PR has been merged

Reply:
Opened a new PR for this change #347